Jennyf/app update

jennyf19 opened the pull request:
Changes include:
> Include XForms UWP dev app for testing in ADAL.

> Remove conditional access flow in test apps, we have [unit test coverage](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/blob/dev/adal/tests/Test.ADAL.NET.Unit/ClaimsChallengeExceptionTests.cs#L43) for this and now we have OBO PM samples for e2e testing. 

> Update ADAL Android test app. The current ClientId was not working, so using the one from the Xamarin Forms app.

Tested UWP and ADAL Android AT interactive and silent flows.

jennyf19 committed:
ADAL test app updates
> Remove conditional access from test apps (we have unit tests for this. we used this before CA was in prod).
> Add UWP XForms platform

jennyf19 committed:
Merge remote-tracking branch 'origin/dev' into jennyf/appUpdate

jennyf19 committed:
missing quotation

bgavrilMS committed:
Add project reference

jennyf19 committed:
update platform parameters

jennyf19 committed:
Merge remote-tracking branch 'origin/dev' into jennyf/appUpdate

jennyf19 committed:
changes to  ADAL test app

henrik-me commented:
Seems like the folder/project name was named wrongly. XFporms ... a p seems to have made it's way in

henrik-me commented:
> {

what is this needed for?

---
Refers to: msal/tests/dev apps/WebTestApp/WebApp/package-lock.json:1 in 10b1399. [](commit_id = 10b1399820431a54b028abd6c7092fcb622b5c03, deletion_comment = False)

jennyf19 commented:
uhhh...i've fixed this with next commit. thanks.

---
In reply to: [414431597](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1211#issuecomment-414431597) [](ancestors = 414431597)

jennyf19 committed:
fix typo

henrik-me commented on adal/devApps/AdalAndroidTestApp/MainActivity.cs:
```diff
@@ -134,34 +133,5 @@ private async void clearCacheButton_Click(object sender, EventArgs e)
             });
         }
-
-        private async void conditionalAccessButton_Click(object sender, EventArgs e)
```
>rivate async void conditionalAccessButton_Click(object sender, EventArgs e) [](start = 9, length = 75)

Are we sure we do not need dev samples for CA? to make testing CA as easy as possible? Do we believe the PM sample is enough also for debugging? #Pending

jennyf19 replied:
I'll sync with @jmprieur on that. i don't think this particular test is of value because the claims are mocked, so it's just using the claims AT overload. We could integrate a true CA e2e test. I'll investigate.

---
In reply to: [211379932](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1211#discussion_r211379932) [](ancestors = 211379932)

henrik-me reviewed (changes requested):
:clock1:

henrik-me commented:
> {

?

---
In reply to: [414432074](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1211#issuecomment-414432074) [](ancestors = 414432074)

---
Refers to: msal/tests/dev apps/WebTestApp/WebApp/package-lock.json:1 in 10b1399. [](commit_id = 10b1399820431a54b028abd6c7092fcb622b5c03, deletion_comment = False)

henrik-me commented:
seems like the folder name wasn't updated

---
In reply to: [414433658](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1211#issuecomment-414433658) [](ancestors = 414433658,414431597)

jennyf19 commented:
@henrik-me I have a [new PR w/the changes](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1212). I have addressed all comments except for follow-up on CA.
